stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -333,6 +335,11 @@ public class ChatClient internal constructor(
         }
     }
 
+    @InternalStreamChatApi
+    public fun containsStoredCredentials(): Boolean {
+        return encryptedPushNotificationsConfigStore.get() != null
```

Comment:
This is no longer `pushNotificationsConfigStore`. Maybe we should rename it to something more meaningful, for example `EncryptedUserConfigStorage`?